audformat/core/scheme.py

```diff
@@ -301,6 +361,33 @@ def _dtype_from_labels(
 
         return dtype
 
+    def _get_labels(
```

Comment:
Why do we need two functions: `_get_labels()` and `_labels_to_list()`? And why is `_get_labels()` sometimes returning a dictionary and sometimes a list?

Reply:
`_get_labels()` is trying to copy the behavior of `Scheme.labels` that we had before, which was returning a list for list labels and a dict for dict labels. I now added the case of a misc table, which is returned as a dict, which we need for applying `map` in `Column`.

At several places we used `list(Scheme.labels)` before. This could in principle be replaced by `list(_get_labels())`, the problem is that inside `replace_labels()` we also need it for labels that are not assigned to the scheme yet, hence `_labels_to_list(labels)` solves this issue.

One solution might be to change `_get_labels(self)` to `_get_labels(self, labels)`, but this also looks strange.

Any other ideas how to make this nicer?

Reply:
> _get_labels() is trying to copy the behavior of Scheme.labels that we had before, which was returning a list for list labels and a dict for dict labels. I now added the case of a misc table, which is returned as a dict, which we need for applying map in Column.

But if I look at the following two tests, wouldn't it be more consistent if we return also a list if the misc table does not have columns?

```
    # replace again with mis table
    scheme.replace_labels('misc')
    assert scheme.labels == 'misc'
    assert scheme._get_labels() == {'spk1': {}, 'spk2': {}}

    # replace with list
    scheme.replace_labels(['spk1', 'spk2'])
    assert scheme.labels == ['spk1', 'spk2']
    assert scheme._get_labels() == ['spk1', 'spk2']
```

Reply:
> Any other ideas how to make this nicer?

Not really.

Reply:
>   But if I look at the following two tests, wouldn't it be more consistent if we return also a list if the misc table does not have columns?

Or as an alternative, always a dictionary. Might be easier to work with if the return value is always of the same type.

Reply:
Ok, cool. I think it's fine to stay with both functions.